### DISCUSSION TITLE
Setting (Protobuf) Schema References

### DISCUSSION THREAD

**Discussion by NgocDungDo** [provectus/kafka-ui#3524] [Q&A] [answered]
Hi I have to Protobuf schemas where one has to reference the other. 
Is there a way to do it with the Kafka-UI?

**Comment by Haarolean** [comment, marked as answer]
Hi, yep:
#3292
[docs](https://github.com/provectus/kafka-ui/blob/master/documentation/guides/Protobuf.md)
